docs/readme/readme_id.md

```
@@ -0,0 +1,226 @@
+<br />
+<img src="../../assets/images/logo-transparent.png" width="80"/>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)